tests/integration/submission/test_upload.py

```diff
@@ -228,24 +231,23 @@ def test_data_file_already_indexed(
     entity1 = assert_single_entity_from_response(resp1)
     assert_positive_response(resp1)
     assert entity1['action'] == 'update'
-
-    # check that record did not change
-    assert record.to_json() == record1.to_json()
+    
+    # check that record did not change, excluding revison
+    record_dict = record.to_json()
+    record1_dict = record1.to_json()
+    record_dict.pop('rev')
+    record1_dict.pop('rev')
+    assert record_dict == record1_dict
 
     # make sure uuid in node is the same as the uuid from index
     # FIXME this is a temporary solution so these tests will probably
     #       need to change in the future
     assert entity['id'] == record1.did
 
 
-@pytest.mark.parametrize('new_urls,id_provided', [
-    (['some/new/url/location/to/add'], False),
-    (['some/new/url/location/to/add'], True),
-    ([DEFAULT_URL, 'some/new/url/location/to/add', 'some/other/url'], False),
-    ([DEFAULT_URL, 'some/new/url/location/to/add', 'some/other/url'], True),
-])
-def test_data_file_update_urls(
-        new_urls, id_provided,
+@pytest.mark.parametrize('id_provided', [False, True])
+def test_data_file_update_patch(
```

Comment:
Comments in docstring and in code does not reflect the changes made, L262 suggests a url change but no url change occurred in code